packages/expo-clipboard/ios/NSAttributedString+utilities.swift

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+import MobileCoreServices
+
+extension NSAttributedString {
+  convenience init(htmlString: String) throws {
+    let initOptions: [DocumentReadingOptionKey: Any] = [
+      .documentType: NSAttributedString.DocumentType.html
+    ]
+    try self.init(data: Data(htmlString.utf8),
+                  options: initOptions,
+                  documentAttributes: nil)
```

Comment:
It's not too long to wrap it (93 chars) 😄
```suggestion
    try self.init(data: Data(htmlString.utf8), options: initOptions, documentAttributes: nil)
```

Reply:
here I agree, at first I inlined `initOptions` that's why I wrapped it earlier 😅